.gitignore

```diff
@@ -60,4 +60,15 @@ vector/**
 InsightCache/
 
 # Python local log file
-py/log.txt
\ No newline at end of file
+py/log.txt
+
+# test folder stuff
+testfolder/project/**
+testfolder/user/**
+testfolder/db/**
+testfolder/model/**
+testfolder/storage/**
+testfolder/vector/**
+testfolder/InsightCache/
+testfolder/mailpit/mailpit.log
+
```

Comment:
Preventing tests from littering changelog. Trying to increase ease of use and usability of testing environment over the coming weeks, since I've neglected the Semoss tests.

Reply:
I think there is an inflection point where we have enough tests and utils, it becomes super simple to write any test and you only have to add one more piece of functionality to the test suite to write a super specific test.